src/map/ability.h

```diff
@@ -45,593 +45,622 @@ enum ADDTYPE
+    ABILITY_EBULLIENCE         = 221,
+    ABILITY_MANIFESTATION      = 222,
+    ABILITY_STRATAGEMS         = 223,
+    ABILITY_VELOCITY_SHOT      = 224,
+    ABILITY_SNARL              = 225,
+    ABILITY_RETALIATION        = 226,
+    ABILITY_FOOTWORK           = 227,
+    ABILITY_DESPOIL            = 228,
+    ABILITY_PIANISSIMO         = 229,
+    ABILITY_SEKKANOKI          = 230,
+    ABILITY_ELEMENTAL_SIPHON   = 232,
+    ABILITY_SUBLIMATION        = 233,
+    ABILITY_ADDENDUM_WHITE     = 234,
+    ABILITY_ADDENDUM_BLACK     = 235,
+    ABILITY_COLLABORATOR       = 236,
+    ABILITY_SABER_DANCE        = 237,
+    ABILITY_FAN_DANCE          = 238,
+    ABILITY_NO_FOOT_RISE       = 239,
+    ABILITY_ALTRUISM           = 240,
+    ABILITY_FOCALIZATION       = 241,
+    ABILITY_TRANQUILITY        = 242,
+    ABILITY_EQUANIMITY         = 243,
+    ABILITY_ENLIGHTENMENT      = 244,
+    ABILITY_AFFLATUS_SOLACE    = 245,
+    ABILITY_AFFLATUS_MISERY    = 246,
+    ABILITY_COMPOSURE          = 247,
+    ABILITY_YONIN              = 248,
+    ABILITY_INNIN              = 249,
+    ABILITY_AVATARS_FAVOR      = 250,
+    ABILITY_READY              = 251,
+    ABILITY_RESTRAINT          = 252,
+    ABILITY_PERFECT_COUNTER    = 253,
+    ABILITY_MANA_WALL          = 254,
+    ABILITY_DIVINE_EMBLEM      = 255,
+    ABILITY_NETHER_VOID        = 256,
+    ABILITY_DOUBLE_SHOT        = 257,
+    ABILITY_SENGIKORI          = 258,
+    ABILITY_FUTAE              = 259,
+    ABILITY_SPIRIT_JUMP        = 260,
+    ABILITY_PRESTO             = 261,
+    ABILITY_DIVINE_WALTZ_II    = 262,
+    ABILITY_FLOURISHES_III     = 263,
+    ABILITY_CLIMACTIC_FLOURISH = 264,
+    ABILITY_LIBRA              = 265,
+    ABILITY_TACTICAL_SWITCH    = 266,
+    ABILITY_BLOOD_RAGE         = 267,
+    ABILITY_IMPETUS            = 269,
+    ABILITY_DIVINE_CARESS      = 270,
+    ABILITY_SANCROSANCTITY     = 271,
+    ABILITY_ENMITY_DOUSE       = 272,
+    ABILITY_MANAWELL           = 273,
+    ABILITY_SABOTEUR           = 274,
+    ABILITY_SPONTANEITY        = 275,
+    ABILITY_CONSPIRATOR        = 276,
+    ABILITY_SEPULCHER          = 277,
+    ABILITY_PALISADE           = 278,
+    ABILITY_ARCANE_CREST       = 279,
+    ABILITY_SCARLET_DELIRIUM   = 280,
+    ABILITY_SPUR               = 281,
+    ABILITY_RUN_WILD           = 282,
+    ABILITY_TENUTO             = 283,
+    ABILITY_MARCATO            = 284,
+    ABILITY_BOUNTY_SHOT        = 285,
+    ABILITY_DECOY_SHOT         = 286,
+    ABILITY_HAMANOHA           = 287,
+    ABILITY_HAGAKURE           = 288,
+    ABILITY_ISSEKIGAN          = 291,
+    ABILITY_DRAGON_BREAKER     = 292,
+    ABILITY_SOUL_JUMP          = 293,
+    ABILITY_STEADY_WING        = 295,
+    ABILITY_MANA_CEDE          = 296,
+    ABILITY_EFFLUX             = 297,
+    ABILITY_UNBRIDLED_LEARNING = 298,
+    ABILITY_TRIPLE_SHOT        = 301,
+	ABILITY_ALLIES_ROLL        = 302,
+	ABILITY_MISER_S_ROLL       = 303,
+	ABILITY_COMPANION_S_ROLL   = 304,
```

Comment:
Looks like some spacing issues in this file too

Reply:
took out tab keys